RELEASE_NOTES.md

```diff
@@ -7,6 +7,7 @@
 ## Upgrading
 
 * `Client.stream()` will now raise an Exception when the connection is lost.
+* `Client.create()` and `Client.list() now take all optional arguments as keyword-only arguments.
```

Comment:
Damn! Missing closing backtick for `Create.list()`.

Reply:
Maybe fix it the next time we update the release notes or before the next release.